modules/tinymce/src/core/demo/html/shadow_dom_closed_demo.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>TinyMCE Shadow DOM Demo Page</title>
```

Comment:
Maybe update the title and h2 heading below, so it's easier to differentiate between the two demos?

Reply:
Done.